tests/framework/db/mssql/MssqlCommandTest.php

```diff
@@ -62,7 +63,7 @@ public function testBindParamValue()
         $command->bindParam(':bool_col', $boolCol);
         $this->assertEquals(1, $command->execute());
 
-        $sql = 'SELECT int_col, char_col, float_col, CONVERT([nvarchar], blob_col) AS blob_col, numeric_col FROM type';
+        $sql = 'SELECT int_col, char_col, float_col, CONVERT([varchar], blob_col) AS blob_col, numeric_col FROM type';
```

Comment:
Leaving `nvarchar` here fails when running tests using the dblib driver under Linux and switching to `varchar` fails when running on sqlsrv on Windows.